web/themes/custom/votegov/templates/taxonomy/taxonomy-term--basics-modules.html.twig

```diff
@@ -4,14 +4,17 @@
  * Default theme implementation to display a taxonomy term: Basics Modules.
  */
 #}
-
 {% extends '@bixaluswds/taxonomy/taxonomy-term.html.twig' %}
 {% set anchor = content.field_anchor_id | field_value | render | clean_class %}
 {% if anchor %}
   {% set attributes = attributes.setAttribute('id', anchor) %}
 {% endif %}
 
+{% set id = term._referringItem.parent.parent.entity.id.value %}
+{% set paragraph_parent = drupal_entity('paragraph', id , 'default') %}
+{% set heading_level = paragraph_parent['#paragraph'].field_heading.value ? '3' : '2' %}
+
 {% block content %}
-  <h3 class="heading--red-underline">{{ content.field_heading | field_value }}</h3>
+  <h{{heading_level}} class="heading--red-underline">{{ content.field_heading | field_value }}</h{{heading_level}}>
```

Comment:
Add a fallback if everything fails for the heading for this line.
I suggest that we use: `<{{heading_level | default('h2')}} ...>`